__tests__/api.test.js

```diff
@@ -109,3 +109,47 @@ describe("api/articles", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  it("should return a status code of 200 and an array", async () => {
+    const response = await request(app)
+      .get("/api/articles/1/comments")
+      .expect(200);
+  });
+  it("should return an array of comment objects with the correct properties", async () => {
+    const { body } = await request(app)
+      .get("/api/articles/1/comments")
+      .expect(200);
+    body.comments.forEach((comment) => {
+      expect(comment).toMatchObject({
+        comment_id: expect.any(Number),
+        body: expect.any(String),
+        article_id: expect.any(Number),
+        author: expect.any(String),
+        votes: expect.any(Number),
+        created_at: expect.any(String),
+      });
+    });
+  });
```

Comment:
Please write another test for a successful response where the article doesn't have any comments yet. This should respond with an empty array.